### DISCUSSION TITLE
How to change the default no data message when the data on the table is empty.

### DISCUSSION THREAD

**Discussion by mersimoski** [KevinVandy/material-react-table#337] [Q&A] [answered]
Currently we have the "no results found"
![image](https://user-images.githubusercontent.com/62712311/214841942-f56f9085-9864-40a5-a6b8-07aaaeb2fb48.png)

But is there any way i can change this message?

**Comment by KevinVandy** [comment, marked as answer]
You can change it in the localization prop

**Reply by mersimoski** [reply]
Thank you, solved it!

**Comment by Hideman85** [comment]
@KevinVandy Could we have a way to customize the rendering?

I need to put a more graphical component with a CTA like below, do you think you could make an option available? 🙏 This would be awesome 💯

```jsx
<MaterialReactTable
  columns={columns}
  data={data}
  // Here I need to be able to customise the empty placeholder
  emptyPlaceholder={<Empty title='No data found' description={<span>No data match the current filters <Button>Reset filters</Button></span>} />}
/>;
```

**Reply by MisAditi** [reply]
@KevinVandy  property emptyaceHolder does not exist on type “IntrinsicAttribute” this is the error I am getting

**Reply by KevinVandy** [reply]
>  @KevinVandy  property emptyaceHolder does not exist on type “IntrinsicAttribute” this is the error I am getting 

That's because that prop/option doesn't exist. Use renderEmptyRowsFallback. Read the docs
